<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NgiiDataUtilsDockWidgetBase</class>
 <widget class="QDockWidget" name="NgiiDataUtilsDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>공간정보 중첩 검사 툴</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="btnAutoDetect">
       <property name="toolTip">
        <string>수주시 받은 기초자료와 납품할 수정성과물 자동비교</string>
       </property>
       <property name="text">
        <string>기초자료/수정성과 비교</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>변화정보내보내기</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnLoadVector">
      <property name="toolTip">
       <string>GeoPackage, Shape, Dxf, 온맵 등 지도 불러오기</string>
      </property>
      <property name="text">
       <string>국토지리정보원 지도 중첩</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnLoadImage">
      <property name="toolTip">
       <string>IMG, Tif 등 영상자료 불러오기</string>
      </property>
      <property name="text">
       <string>국토지리정보원 영상 중첩</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btnLoadTms">
      <property name="toolTip">
       <string>국가인터넷지도, 다음지도, 네이버지도, OSM 불러오기</string>
      </property>
      <property name="text">
       <string>인터넷지도 중첩</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btnLoadBaseMap">
      <property name="toolTip">
       <string>국토기본정보 최신 자료 불러오기</string>
      </property>
      <property name="text">
       <string>국토기본정보 중첩</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QProgressBar" name="prgSub">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>3</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>3</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QProgressBar" name="prgMain">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>5</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>5</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lblStatus">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>진행상황 표시</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPlainTextEdit" name="editLog">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnReportError">
        <property name="toolTip">
         <string>발견된 오류사항을 캡처하고 내용 기록</string>
        </property>
        <property name="text">
         <string>오류사항 기록</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMakeReport">
        <property name="toolTip">
         <string>오류보고를 PDF 파일로 생성</string>
        </property>
        <property name="text">
         <string>오류보고 생성</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>245</width>
         <height>257</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
